### ISSUE TITLE
Feature: Add a tasks screen

### ISSUE THREAD

**Issue by KeziahMoselle** [KeziahMoselle/tempus#78]
## Solution
Add a screen where the user can create tasks

A task can be :
- Unchecked (default)
- Checked (done)

Show the task currently being worked on on the counter screen

**Comment by ferfabricio** [comment]
I've forked this repo and will try implement this screen.

**Comment by KeziahMoselle** [comment]
Great !
If you have any questions don't hesitate !
I don't know where to put this screen so yeah :/ if you have ideas before implementing this I think we should discuss it first

Also done tasks should be hidden

**Comment by ferfabricio** [comment]
What if we add one more button on the footer with this tasklist?

**Comment by KeziahMoselle** [comment]
Yeah I think it's a good idea, but the centering of the icons will be broken, for example if we add a third icon the counter setting icon will be placed in the center and it will clash with the start/stop overlapping icon

**Comment by KeziahMoselle** [comment]
Also the tasks should be temporary, I think it will help the users to focus on 1 or 2 tasks per session, so no need to store them 👍

**Comment by ferfabricio** [comment]
I was thinking in put one Icon centered on left and two on right, but I will make some tests of how this will be in shows and update in this thread

**Comment by KeziahMoselle** [comment]
Okay !
